compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFactory.java

```diff
@@ -363,6 +367,15 @@ public BallerinaVariableSymbol createVariableSymbol(BVarSymbol symbol, String na
                 .build();
     }
 
+    /**
+     * @param symbol {@link BVarSymbol} to convert
+     * @return {@link BallerinaTupleMemberSymbol} generated
+     */
+    public BallerinaTupleMemberSymbol createTupleMember(BVarSymbol symbol) {
```

Comment:
Does this have to be public? Please update docs to include a description if so.

Reply:
Not needed to be public, changed